sapl/materia/views.py

```diff
@@ -1,4 +1,5 @@
 from datetime import datetime, date
+from django.utils import timezone
```

Comment:
Use `from datetime import datetime`. Estamos com um PR para tentar mudar de `datetime` para `timezone`, mas isso talvez demore, **se for possível**. Modifique isso.